index.d.ts

```diff
@@ -139,6 +184,10 @@ declare module 'react-native-twilio-video-webrtc' {
 
     getStats: () => void;
 
+    requestStats: (intervalMs: number) => void;
+
+    cancelStatsRequest: () => void;
```

Comment:
[Minor] consider more obvious names.
Eg: `startSendingStats` and `stopSendingStats`